pom.xml

```diff
@@ -735,6 +736,33 @@ THE SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <effort>Max</effort>
+          <threshold>High</threshold>
+          <!--Excludes file is located on the top level-->
+          <excludeFilterFile>../src/findbugs/findbugs-excludes.xml</excludeFilterFile>
```

Comment:
But this file isn't there for the build of this pom (and even invoking findbugs on a project of type pom is well questionable....)

And it is bound to the lifecycle here as it is bound to the verify stage... and in the `<plugins>` section not the `<pluginManagement>` section.